### PR TITLE
Refactor to replace helm chart with go installer

### PR DESCRIPTION
**Pull request by aledbf**
## Description

Requires https://github.com/gitpod-io/gitpod/pull/6816
Requires https://github.com/gitpod-io/gitpod/pull/6912

fixes https://github.com/gitpod-io/gitpod-eks-guide/issues/25

```release-note
Refactor to replace helm chart with go installer
```

### CODE REVIEW

File: lib/gitpod.ts [hunk 1]
```diff
@@ -1,85 +1,34 @@
...
-
-export interface GitpodProps extends cdk.StackProps {
+export interface GitpodProps extends StackProps {
     domain: string
 
     certificateArn?: string
 
-    database: Database
-    registry: Registry
+    //database: Database
```
Comment (metcalfc) [inline lib/gitpod.ts:21]: Are these just dead code now or are we leaving a todo for future us?

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,8 @@
-FROM alpine:3.14
+ARG GITPOD_VERSION="aledbf-insts3.1"
```
Comment (metcalfc) [inline Dockerfile:1]: Do we need to document the path to get these on the standard versions?

**Review by metcalfc — COMMENTED** [review]
This is generally LGTM, three things I'm checking on: 

1. How are we thinking on version deltas to other cloud stacks?
2. The commented-out bits should just go?
3. Whats the path to get onto a gitpod version (doesn't need to block, track in an issue is fine)

**Review by MrSimonEmms — APPROVED** [review]
Should change the `GITPOD_VERSION`, but fine other than that

File: Dockerfile (hunk 2 above)
Comment (aledbf) [reply Dockerfile]: fixed

File: lib/gitpod.ts (hunk 1 above)
Comment (aledbf) [reply lib/gitpod.ts]: removed

**Pull request merged**